### ISSUE TITLE
Add version to meta-pack, save version in JSON

### ISSUE THREAD

**Issue by nicebread** [nicebread/RESQUE#80]

**Comment by alpkaanaksu** [comment]
We always had it?

**Comment by nicebread** [comment]
But the version of the meta-pack is not saved in the exported json:

```
[
  {
    "type": "meta",
    "queryConfig": {
      "global": {}
    },
    "position": 0,
    "date_created": 1710145525582,
    "date_modified": 1710145531542,
    "LastName": "Test",
    "FirstName": "Test",
    "YearPhD": "2123",
    "AcademicAgeBonus": "0",
    "RaterType": "Applicant",
    "date_exported": 1710145536841
  },
  {
    "type": "pub",
    "version": "0.3.1",
    "date_added": 1710145531542,
```

**Comment by alpkaanaksu** [comment]
Ok, I see the problem now. Fixed here: a7edea2f06548feb03edb74113179ce56c763aee

```json
[
  {
    "type": "meta",
    "version": "0.2.0",
    "date_created": 1710157778773,
    "queryConfig": {
      "global": {
        "mode": "preview-single"
      }
    },
    "position": 0,
    "date_modified": 1710157819478,
    "LastName": "Test",
    "FirstName": "Test",
    "YearPhD": "2123",
    "AcademicAgeBonus": "0",
    "ORCID": ".",
    "RaterType": "Applicant",
    "date_exported": 1710157828288
  },
  {
    "type": "pub",
    "version": "0.3.1",
    "date_added": 1710157787366,
```
